### PR TITLE
update broken link

### PR DESCRIPTION
**Pull request by mirnawong1**
this pr fixes a link typo on the same page as if someone clicked on the 'below' link, it would take the user to a 'page not found. it removes the extra `[` and `-` as well.

### CODE REVIEW

**Review by leighstaub — APPROVED** [review]
Thanks @mirnawong1 !

**Comment by mirnawong1** [comment]
thanks so much @leighstaub ! am i ok merging this? i know the vercel deployment failed

**Comment by leighstaub** [comment]
@mirnawong1 Ah, good question -- I didn't realize that. Honestly, I don't know what that failure means. @john-rock do you know if that's required?

**Comment by john-rock** [comment]
> @mirnawong1 Ah, good question -- I didn't realize that. Honestly, I don't know what that failure means. @john-rock do you know if that's required?

Hey y'all, @mirnawong1 I beleive the build failed becuase it seemed like you didn't have permissions to create builds on this repo. I double checked the build logs and confirmed there are no builds triggered by you in this repo.

@mirnawong1 I just updated your permissions. When you have a moment, can you try to push a new commit and see if the build gets triggered?

File: website/docs/time_off.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ When in doubt, please discuss with your manager about your plans.
 
 Note that because, in the United States, your time off is not accrued, you will not be paid out for any unused time offered under this policy when your employment with dbt Labs ends.
 
-For international employees, see [below]([#-international-considerations-for-accrued-time-and-pay-out-upon-termination-with-the-company) for respective details.
+For international employees, see [below](#international-considerations-for-accrued-time-and-pay-out-upon-termination-with-the-company) for respective details.
 
 #### 💪 Best practices for being out of the office
```
Comment (mirnawong1) [inline website/docs/time_off.md:65]:
```suggestion
 
```

**Comment by mirnawong1** [comment]
you got it, thank you all!

**Comment by mirnawong1** [comment]
woohoo! merging now, thank you team!

**Pull request merged**